modape/scripts/modis_smooth.py

```diff
@@ -217,13 +217,13 @@ def _worker(rawfile: str,
         smt_h5.create()
 
     if last_collected is not None:
-        last_collected_infile = smt_h5.last_collected
+        last_collected_in_rawfile = smt_h5.last_collected_in_raw
```

Comment:
depending if the naming changes, it has to change here too. But VSC will take care of that I'm assuming

Reply:
See latest commits; renamed